src/promptflow/promptflow/_utils/multimedia_utils.py

```diff
@@ -263,3 +275,11 @@ def resolve_image_path(input_dir: Path, image_dict: dict):
             if resource == "path":
                 image_dict[key] = str(input_dir / image_dict[key])
     return image_dict
+
+
+class InvalidImageInput(ValidationException):
+    pass
+
+
+class LoadMultimediaDataError(UserErrorException):
```

Comment:
I think current convention is to define error codes in a centralized file?

Reply:
Defined the errors in the `_utils._errors.py`.